caikit/core/task.py

```diff
@@ -359,6 +377,12 @@ def run_bidi_stream(raw_documents: DataStream[caikit.interfaces.nlp.RawDocument]
             task, which all modules' streaming-output inference methods must return. This must be
             in the form Iterable[T].
 
+        visible (bool): If this task should be exposed to the end user in documentation or if
+          it should only be used internally
+
+        extra_openapi_schema (Optional[Dict[str, Any]]): Any additional openapi schema that should
+          be included in the documentation for this task
```

Comment:
NIT to GitHub! The "suggestion" feature needs the ability to set a word-wrap. Probably need to verify this

```suggestion
        metadata (Optional[Dict[str, Any]]): Any additional interface
            information (e.g. OpenAPI schema) that should be included in the
            documentation for this task
```